src/WaterNetwork.jl

```diff
@@ -137,8 +141,11 @@ function constraintoffset_waternetwork_outflows(m::Model)
     end
 
     function generate(gg, tt)
-        # Determine number of gauges in county
-        b[gg, tt]
+        if get(config, "proportionnaturalflowforenvironment", nothing) == nothing
+            b[gg, tt]
+        else
+            (1-config["proportionnaturalflowforenvironment"])*b[gg, tt]
+        end
```

Comment:
I split this out as two separate generate functions in `scenarios`, so we don't have to make the check every time.

Reply:
It is now separated in two different constraints offset functions, the original one in WaterNetwork, and the environmental part in EnvironmentalDemand.jl